<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>magacinAzuriraj</class>
 <widget class="QDialog" name="magacinAzuriraj">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>993</width>
    <height>515</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Aplikacija Pekara | Ažuriranje magacina</string>
  </property>
  <widget class="QLabel" name="naslov">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>20</y>
     <width>321</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Ažuriranje sirovine u magacinu:</string>
   </property>
  </widget>
  <widget class="QLabel" name="labela_ID">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>100</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Šifra sirovine:</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBoxID">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>100</y>
     <width>91</width>
     <height>22</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLabel" name="jedMereLabela">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>340</y>
     <width>131</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Jedinica mere:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="azurirajDugme">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>420</y>
     <width>171</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Ažuriraj</string>
   </property>
  </widget>
  <widget class="QPushButton" name="dugmeIzlazAz">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>420</y>
     <width>171</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Izlaz</string>
   </property>
  </widget>
  <widget class="QLabel" name="KolicinaLabela_az">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>220</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Količina:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_kolicina_az">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>210</y>
     <width>321</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QComboBox" name="magacinIzbor_az">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>330</y>
     <width>321</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <item>
    <property name="text">
     <string>Gram</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Kilogram</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Komad</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Tona</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="NazivLabela">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>160</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Naziv:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_naziv_az">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>150</y>
     <width>321</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLabel" name="KolicinaLabela_az_2">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>280</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Nabavna cena:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_nbCena">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>270</y>
     <width>321</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
